src/client/plugins/vue-material.scss

```diff
@@ -6,7 +6,7 @@
 
 @include md-register-theme("default", (
   primary: #008FC5, // The primary color of your brand
-  accent: #709DB1 // The secondary color of your brand
+  accent: #FF7200 // The secondary color of your brand
```

Comment:
this is not really the accent color, its more a highlight color or something. Check if you can add an extra property here instead of accent and restore accent,